### PR TITLE
[WIP] Automate versioning and sanitize recursor distribution

### PR DESCRIPTION
**Pull request by pieterlexis**
As there are different version numbers for the auth and recursor releases, make should create correct tarballs. This PR keeps track of all the things that have to be done to make this happen.

### CODE REVIEW

**Comment by pieterlexis** [comment]
We can probably do without the `.version` file if we 'just' define the version as a var in `configure.ac` so it carries over to `configure` and we don't have to distribute the `.version` file.

File: build-scripts/build-auth-rpm [hunk 1]
```diff
@@ -0,0 +1,567 @@
...
+Group: System Environment/Daemons
+Requires: %{name}%{?_isa} = %{version}-%{release}
+BuildRequires: openldap-devel
+%global backends %{backends} ldap
+
+%description backend-ldap
+This package contains the ldap backend for %{name}
+
+%package backend-lua
+Summary: LUA backend for %{name}
```
Comment (Habbie) [inline build-scripts/build-auth-rpm]: Lua ;)

**Comment by Habbie** [comment]
Other than the simplebind stuff which keeps pissing people off, this looks fine to me.

File: build-scripts/build-auth-rpm (hunk 1 above)
Comment (jpmens) [inline build-scripts/build-auth-rpm]: LDAP ;)

**Comment by pieterlexis** [comment]
@zeha @mind04 @cmouse if you'd be so kind: apart from the readme, this is ready for review

**Comment by zeha** [comment]
I've only had a quick look, but that LGTM

**Comment by cmouse** [comment]
Imo it's not normal in rhel based systems that services start themselves after install, or did I misread the pr?

**Comment by cmouse** [comment]
@pieterlexis did you check licenses on those translation files?

**Comment by pieterlexis** [comment]
The translation files are GPL-2 according to debian/copyright

**Comment by rubenk** [comment]
@cmouse Regarding chkconfig, the .spec only runs chkconfig --add which adds the service. Since the init script contains `# chkconfig: - 80 75` It doesn't enable the service.

**Pull request merged**